Command/ReturnMessage/example1.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
See previous comments regarding naming and properties

Reply:
Changed the name of the examples to "example-***.json"